source/sync/app-builder/local-to-sync.txt

```diff
@@ -125,3 +110,13 @@ App Logic
 
 #. On each subsequent app load, check if the local realm has been
    deleted.
+
+.. figure:: /images/local-to-sync-data-copy.png
```

Comment:
This is so nice to have. I need to add links to this section in the relevant parts of the JS and Flutter SDKs.